### ISSUE TITLE
Add bazel compatibility table

### ISSUE THREAD

**Issue by igorgatis** [bazelbuild/rules_kotlin#185]
I'm using bazel version 0.22.0. I have no clue which version of rules_kotlin to use.

rules_scala has this table:

![image](https://user-images.githubusercontent.com/1269388/57801899-3485e500-772b-11e9-8c82-74528dfdd497.png)

It would be very useful to have something similar in kotlin.

**Comment by igorgatis** [comment]
Sounds like this table would be useful for #182

**Comment by igorgatis** [comment]
And in #165

**Comment by cgruber** [comment]
Yeah, in my fork I've started to name releases after versions of bazel it's intended to work with.  E.g. `legacy-modded-0_26_1-01`. It's not perfect, but hopefully conveys part of this.
